src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -158,10 +159,14 @@ public final boolean perform(final AbstractBuild<?, ?> build, final Launcher lau
 
         String commentAction = "none";
         if (runHarbormaster) {
-            logger.info("harbormaster", "Sending build result to Harbormaster with PHID '" + phid + "', success: " + harbormasterSuccess);
+            logger.info("harbormaster", "Sending Harbormaster BUILD_URL via PHID: " + phid);
+            Task.Result sendUriResult = new SendHarbormasterUriTask(logger, diffClient, phid, environment.get("BUILD_URL")).run();
```

Comment:
We're grabbing the BUILD_URL environment variable above on line 144 -- let's move it out into a final String variable to dedup the logic for fetching it